.circleci/config.yml.j2

```
@@ -11,6 +11,9 @@ workflows:
           airflow_version: {{ airflow_version }}
           distribution_name: {{ distribution }}
           docker_repo: {{ docker_repo }}
+          {%- if "dev" in airflow_version %}
+          extra_args: "--build-arg VERSION=$(curl https://pip.astronomer.io/simple/astronomer-certified/latest-{{ airflow_version | replace('.dev', '') }}.build)"
```

```suggestion
          extra_args: "--build-arg VERSION===$(curl https://pip.astronomer.io/simple/astronomer-certified/latest-{{ airflow_version | replace('.dev', '') }}.build)"
```

(no need to quote `=` as it's not a special shell character.

---

I moved '==' inside the Dockerfile

---

Actually moved that logic inside Dockerfile